src/pytestqt/plugin.py

```diff
@@ -226,18 +227,4 @@ def pytest_configure(config):
 
     if config.getoption("qt_log") and config.getoption("capture") != "no":
         config.pluginmanager.register(QtLoggingPlugin(config), "_qt_logging")
-
-    qt_api.set_qt_api(config.getini("qt_api"))
-
-
-def pytest_report_header():
```

Comment:
This should definitely stay - it's very useful information for test runs, and it missing also currently breaks a lot of self tests.